packages/@okta/vuepress-site/docs/guides/customize-authz-server/create-rules-for-policy/index.md

```diff
@@ -32,10 +16,33 @@ Then the access token that is granted has a lifetime of, for example, one hour.
 
 You can also use rules to restrict grant types, users, or scopes. For example, you could prevent the use of all scopes other than `openid` and `offline_access` by only creating rules that specifically mention those two scopes. This means you would have to:
 
-1. Not create any rules that match "any scopes", and
-2. Ensure that all of your rules only match the `openid` and/or `offline_access` scopes.
+* Not create any rules that match "any scopes", and
+
+* Ensure that all of your rules only match the `openid` and/or `offline_access` scopes.
```

Comment:
Or periods.

Could you rewrite these two bullets as a sentence?
> This means you can't create any rules that match "any scopes" and you also ensure that all of your rules only match the `openid` or `offline_access` scopes.

Reply:
I like how it is separated out. I've tried a different format.